mod-users/src/main/resources/domain/mod-users/features/users.feature

```diff
@@ -0,0 +1,158 @@
+Feature: Users tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def status = true
+    * def lastName = call random_string
+    * def firstName = call random_string
+    * def username = call random_string
+    * def email = 'abc@pqr.com'
+
+  Scenario: Create a new User with PatronGroup.
+    * def username = call random_string
+    * def barcode = call random_numbers
+    * def uuid = call uuid1
+    * def createUserResponse = call read('classpath:domain/mod-users/features/util/initData.feature@PostPatronGroupAndUser')
```

Comment:
Why define a variable to capture the response when the test does not inspect it in any way?

(similarly for the similar examples below)